pkg/aws/eni/instances.go

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2019 Authors of Cilium
+// Copyright 2017 Lyft, Inc.
```

Comment:
Where is this coming from? Was the file originally copied from somewhere else?

Reply:
Hmm, i have been filling in Lyft copyrights see:

https://cilium.slack.com/archives/C2B917YHE/p1627401901084700?thread_ts=1626209673.481600&channel=C2B917YHE&message_ts=1627401901.084700

tho, id expect to see a diff here showing the Lyft copyright being moved. Ill double check this one.

Reply:
Oh, I see... That'll be quite easy to miss :cry: